### PR TITLE
Mark more text for localization

### PR DESCRIPTION
**Pull request by quiple**

### CODE REVIEW

**Comment by quiple** [comment]
I think it should be fine to merge, what do you think?

**Comment by quiple** [comment]
@ansh

File: src/locale/locales/ko/messages.po
Comment (ansh) [inline src/locale/locales/ko/messages.po:12]: Please add your name here and in `Language-Team` for credits!

File: src/locale/locales/ko/messages.po
Comment (ansh) [inline src/locale/locales/ko/messages.po:18]: Why has this been commented out?

File: src/locale/locales/ko/messages.po
Comment (quiple) [reply src/locale/locales/ko/messages.po:12]: > Please add your name here and in `Language-Team` for credits!

Yes, It's done.

File: src/locale/locales/ko/messages.po
Comment (quiple) [reply src/locale/locales/ko/messages.po:1934]: I didn't translate "See what's next" and "What's up?" because they don't translate naturally. Do you know of a proper translation?

Comment (lens0021) [reply src/locale/locales/ko/messages.po:1942]: Nope, then LGTM!

**Comment by ansh** [comment]
Great thanks for confirming these translations are correct @lens0021 

@quiple Can you fix the linting errors so I can merge this PR?

File: src/lib/strings/time.ts
Comment (ansh) [inline src/lib/strings/time.ts:22]: Is there another short hand way of saying "seconds" in Korean? For example, this text could be `10s`. Is there a reason to substitute the `s` in Korean?

Comment (quiple) [reply src/lib/strings/time.ts:22]: In Korea, time is typically written in Hangul, so s is replaced with "초", m with "분", and h with "시간". This will also change in Japanese and Chinese to their own language.

Comment (quiple) [reply src/lib/strings/time.ts:22]: In CJK, `10s` is not usally. Replacing time markers like `s` in CJK with your own language is the same for most websites, including Twitter, Instagram, and Facebook.

File: src/locale/i18n.ts
Comment (quiple) [inline src/locale/i18n.ts:12]: I rearranged this alphabetically.

File: src/view/com/util/numeric/format.ts
Comment (quiple) [inline src/view/com/util/numeric/format.ts:1]: It seems to be working fine, but I'd like to know if it's okay as is. I got a linting error, so I fixed the first letter of `formatCount` to be capitalised.

File: src/App.native.tsx
Comment (ansh) [inline src/App.native.tsx:57]: I don't think we would want this string to be localized as localizing it will have a lot of performance implications since this is the root component and it'll be re-rendered

Comment (ansh) [reply src/App.native.tsx:57]: So I think let's remove this localization

File: src/lib/sharing.ts
Comment (ansh) [inline src/lib/sharing.ts:16]: This function is not a react hook and therefore useLingui cannot be used here. The `t` macro is better suited.

File: src/locale/i18n.ts
Comment (ansh) [reply src/locale/i18n.ts:12]: Sure

File: src/state/queries/preferences/moderation.ts
Comment (ansh) [inline src/state/queries/preferences/moderation.ts:88]: @pfrazee More of a moderation question -- do we want to risk our intention of these labels being mistook by allowing the title and subtitle to be localized strings?

File: src/view/com/auth/SplashScreen.web.tsx
Comment (ansh) [inline src/view/com/auth/SplashScreen.web.tsx:108]: Just because these are business links we probably don't want this text localized

File: src/view/com/auth/SplashScreen.web.tsx
Comment (ansh) [inline src/view/com/auth/SplashScreen.web.tsx:113]: We probably want to revert this as wella

File: src/view/com/auth/SplashScreen.web.tsx
Comment (ansh) [inline src/view/com/auth/SplashScreen.web.tsx:118]: We probably want to revert this as well

File: src/view/com/auth/create/Step1.tsx
Comment (ansh) [inline src/view/com/auth/create/Step1.tsx:61]: Just because this is so crucial, we probably don't want to localize this string

File: src/view/com/auth/create/StepHeader.tsx
Comment (ansh) [inline src/view/com/auth/create/StepHeader.tsx:15]: We can wrap both these strings in the `<Trans>` macro

Comment (ansh) [reply src/view/com/auth/create/StepHeader.tsx:15]: Instead of using `_(msg``)`

File: src/view/com/composer/Composer.tsx
Comment (ansh) [inline src/view/com/composer/Composer.tsx:343]: Why was `(verb)` added here?

File: src/view/com/modals/ChangeHandle.tsx
Comment (ansh) [inline src/view/com/modals/ChangeHandle.tsx:396]: @pfrazee Should we localize this?

File: src/view/com/modals/DeleteAccount.tsx
Comment (ansh) [inline src/view/com/modals/DeleteAccount.tsx:88]: Does this mess up the spacing? Did you check on a real device?

File: src/view/com/notifications/FeedItem.tsx
Comment (ansh) [inline src/view/com/notifications/FeedItem.tsx:243]: Why was this function capitalized?

File: src/view/com/post-thread/PostThreadItem.tsx
Comment (ansh) [inline src/view/com/post-thread/PostThreadItem.tsx:405]: Why was this function capitalized?

File: src/view/com/profile/ProfileHeader.tsx
Comment (ansh) [inline src/view/com/profile/ProfileHeader.tsx:241]: Why was this function capitlalized? Usually that is reserved for React Components.

File: src/view/com/profile/ProfileHeader.tsx
Comment (ansh) [inline src/view/com/profile/ProfileHeader.tsx:441]: Why was this function capitlalized? Usually that is reserved for React Components.

File: src/view/com/util/forms/PostDropdownBtn.tsx
Comment (ansh) [inline src/view/com/util/forms/PostDropdownBtn.tsx:144]: Why was this function capitlalized? Usually that is reserved for React Components.

File: src/view/com/util/numeric/format.ts
Comment (ansh) [inline src/view/com/util/numeric/format.ts:4]: Why was this function capitalized? Usually that is reserved for React Components.

File: src/view/screens/ProfileFeed.tsx
Comment (ansh) [inline src/view/screens/ProfileFeed.tsx:252]: Why was this function capitalized? Usually that is reserved for React Components

File: src/lib/sharing.ts
Comment (ansh) [inline src/lib/sharing.ts:15]: Why was this function capitalized? Usually that is reserved for React Components.

**Review by ansh — CHANGES_REQUESTED** [review]
I've requested a few changes. 

It would be better to separate out the marking more text for localization part of this PR and the Korean Localization part of this PR into different PRs, if possible.

File: src/App.native.tsx
Comment (quiple) [reply src/App.native.tsx]: Okay, I understood.

File: src/view/com/auth/SplashScreen.web.tsx
Comment (quiple) [reply src/view/com/auth/SplashScreen.web.tsx:108]: But in `HomeLoggedOutCTA.tsx`, it was already localized.

File: src/view/com/composer/Composer.tsx
Comment (quiple) [reply src/view/com/composer/Composer.tsx:343]: In English, they are both same "Post", but some other languages use different noun and verb.

I've added `(verb)` to distinguish between them, but if there's another way to do it, I'd like to change it.

File: src/view/com/modals/ChangeHandle.tsx
Comment (quiple) [reply src/view/com/modals/ChangeHandle.tsx:396]: Originally, the account handle was pinned after, but due to localization, I needed to include it in the `<Trans>` to allow us to reverse the order.

File: src/view/com/modals/DeleteAccount.tsx
Comment (quiple) [reply src/view/com/modals/DeleteAccount.tsx:88]: Originally, the account handle was pinned after, but due to localization, I needed to include it in the `<Trans>` to allow us to reverse the order.

File: src/lib/sharing.ts
Comment (quiple) [reply src/lib/sharing.ts]: Thanks, I fixed this.

File: src/view/com/auth/create/StepHeader.tsx
Comment (quiple) [reply src/view/com/auth/create/StepHeader.tsx]: Thanks, I fixed it.

File: src/view/com/notifications/FeedItem.tsx
Comment (quiple) [reply src/view/com/notifications/FeedItem.tsx:243]: I changed it to capitalized because I got a linting error after localization, if there's another way to do it, I'd appreciate it.

File: src/view/com/profile/ProfileHeader.tsx
Comment (quiple) [reply src/view/com/profile/ProfileHeader.tsx]: It was reverted, thanks.

File: src/view/com/util/forms/PostDropdownBtn.tsx
Comment (quiple) [reply src/view/com/util/forms/PostDropdownBtn.tsx]: It was reverted, thanks.

File: src/view/screens/ProfileFeed.tsx
Comment (quiple) [reply src/view/screens/ProfileFeed.tsx]: It was reverted, thanks.

File: src/lib/sharing.ts
Comment (quiple) [reply src/lib/sharing.ts]: It was reverted. After changing to the `t` macro, I no longer get the linting error, thanks.

File: src/view/com/composer/Composer.tsx
Comment (quiple) [reply src/view/com/composer/Composer.tsx:343]: In English, they both appear as "Post", just like they always have.

File: src/view/com/notifications/FeedItem.tsx
Comment (quiple) [reply src/view/com/notifications/FeedItem.tsx]: I resolve the linting error. It reverted to lowercase.

File: src/view/com/post-thread/PostThreadItem.tsx
Comment (quiple) [reply src/view/com/post-thread/PostThreadItem.tsx]: I resolve the linting error. It reverted to lowercase.

File: src/view/com/profile/ProfileHeader.tsx
Comment (quiple) [reply src/view/com/profile/ProfileHeader.tsx]: I resolve the linting error. It reverted to lowercase.

File: src/view/com/util/numeric/format.ts
Comment (quiple) [reply src/view/com/util/numeric/format.ts]: I resolve the linting error. It reverted to lowercase.

File: src/view/com/modals/DeleteAccount.tsx
Comment (quiple) [reply src/view/com/modals/DeleteAccount.tsx:88]: "Delete Account "(account name)"" should be localized to Korean as ""(account name)" 계정 삭제".

File: src/view/com/auth/SplashScreen.web.tsx
Comment (quiple) [reply src/view/com/auth/SplashScreen.web.tsx:113]: But in HomeLoggedOutCTA.tsx, it was already localized.

File: src/view/com/auth/SplashScreen.web.tsx
Comment (quiple) [reply src/view/com/auth/SplashScreen.web.tsx:118]: But in HomeLoggedOutCTA.tsx, it was already localized.

File: src/view/com/util/numeric/format.ts
Comment (quiple) [reply src/view/com/util/numeric/format.ts:1]: I resolve the linting error. It reverted to lowercase.

**Comment by quiple** [comment]
I've fixed a linting error in `shareUrl` and `formatCount`, reverting the first letter back to lowercase. We've also made other changes as requested, but I'd like to discuss another requests further.

I've also created another branch for Korean localization and will PR it again later.

**Comment by ansh** [comment]
Thanks! #2395 has been merged separately.

**Comment by pfrazee** [comment]
Whew this is intense! I'm somewhat inclined to get all of the in-flight translations merged, then pull this in

**Comment by pfrazee** [comment]
There are also some standing issues that ansh has already remarked upon that do need to be addressed before it can merge

**Comment by pfrazee** [comment]
But thank you for all this work @quiple !

File: src/view/com/composer/Composer.tsx
Comment (quiple) [reply src/view/com/composer/Composer.tsx:347]: As mentioned in #2446, this appears to be an issue with other languages as well, including Portuguese. I think it needs to add `(verb)` or some other text after the `msgid` to distinguish them.

File: src/Navigation.tsx
Comment (ansh) [inline src/Navigation.tsx:418]: We can remove this

File: src/Navigation.tsx
Comment (ansh) [inline src/Navigation.tsx:80]: We can remove this import as well since we are directly using the `msg` macro

File: src/lib/api/index.ts
Comment (ansh) [inline src/lib/api/index.ts:80]: We should probably not localize this

Comment (ansh) [reply src/lib/api/index.ts:80]: Or anything in this file

File: src/lib/strings/time.ts
Comment (ansh) [reply src/lib/strings/time.ts:24]: I'll let @pfrazee take the call here

File: src/locale/locales/uk/messages.po
Comment (ansh) [inline src/locale/locales/uk/messages.po:1]: Can you delete all the `.po` files just so we don't have a lot of merge conflcits??

File: src/view/com/modals/InviteCodes.tsx
Comment (ansh) [inline src/view/com/modals/InviteCodes.tsx:56]: Double import by mistake -- can delete

File: src/view/com/modals/ServerInput.tsx
Comment (ansh) [inline src/view/com/modals/ServerInput.tsx:94]: We don't want to localize a URL

File: src/view/com/modals/report/ReasonOptions.tsx
Comment (ansh) [inline src/view/com/modals/report/ReasonOptions.tsx:13]: @pfrazee Do we want to localize these strings?

File: src/view/screens/Debug.tsx
Comment (ansh) [inline src/view/screens/Debug.tsx:185]: We can remove these are no point in localizing the `Debug.tsx` file

**Review by ansh — CHANGES_REQUESTED** [review]
Most of the changes from this PR were merged with #2440 

So I'll leave it to @pfrazee if this needs to be merged.

But the best solution would be to create a new clean PR with the few leftover strings. Or at the very least delete the generated `.po` files from this PR so it can be reviewed in depth again.

I also left some comments about some text that we might not want to localize.

In any case, really appreciate your contributions @quiple . Most of them were merged in a different PR but thank you very much nonetheless!!

**Comment by quiple** [comment]
@ansh Thanks for review! I'll request a new PR later.

File: src/view/screens/Debug.tsx
Comment (quiple) [reply src/view/screens/Debug.tsx:185]: It looks like there are other localized strings already in the `Debug.tsx` file, does that make any difference?

File: src/view/com/modals/ServerInput.tsx
Comment (quiple) [reply src/view/com/modals/ServerInput.tsx:94]: I included it in the localization because of `e.g.`. Is it okay to wrap the URL in `${}`?

**Pull request closed without merging**
